docusaurus.config.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
   tagline: 'The tagline of my site',
   url: 'https://docs.nimbella.io',
   baseUrl: '/',
-  onBrokenLinks: 'throw',
+  onBrokenLinks: 'warn',
```

Comment:
Is this a temporary relaxation?

Reply:
If you're asking if I intend to commit it, yes.   How permanent it is depends on the experience of others.   My experience is that the automated build process sparked by pushing commits gives out errors that make no sense (because the complained-about links are not actually broken).    As we gain greater understanding, if we believe we can trust that all the hard errors issued by the build really should be there, then we should restore the flag to its former value.